packages/vitest/src/node/config.ts

```diff
@@ -175,6 +175,9 @@ export function resolveConfig(
       resolved.server.deps![option] = resolved.deps[option] as any
   })
 
+  if (options.exclude)
+    resolved.exclude.push(...options.exclude)
```

Comment:
I don't think this works as you expect. On line 84 `resolved` is created - you can see that options from CLI will just override user config.

This is why I proposed storing the value in something like `cliExcludes` and remove `excludes` before merging into resolved

Reply:
Solved by 9d382a3